01_fib/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(limit int) {
+	firstNo, secondNo := 0, 1
+	var finalString string
+	for i := 0; i < limit; i++ {
+		finalString += strconv.Itoa(firstNo)
+		firstNo, secondNo = secondNo, secondNo+firstNo
+	}
+	fmt.Fprintln(out, finalString)
```

Comment:
This doesn't create the requested output (newline separated).
Try:
```go
	a, b := 1, 1
	for i := 1; i <= n; i++ {
		fmt.Fprintln(out, a)
		a, b = b, a+b
	}
```

Reply:
It's better to decouple fibonacci sequence calculation from IO. https://github.com/anz-bank/go-training/issues/37

Reply:
added the new lines as per the formatting requirements